### ISSUE TITLE
OCD area object pattern rotation not applied after import

### ISSUE THREAD

**Issue by dg0yt** [OpenOrienteering/mapper#1926]
### Steps to reproduce
1. Import an OCD 8/9/10/11/12 file with rotated object patterns, e.g. vineyard
2. Patterns are not shown with the user defined rotation.
3. Activate "adjustable per object" for the fill patterns of the symbol.
4. Now the patterns are shown correctly

### Actual behaviour
Rotations are loaded but not applied, due to incorrect symbol configuration. They are lost on export.

### Expected behaviour
The symbol properties must be set up correctly.

### Configuration
Mapper Version: v0.9.5
Operating System: any

### More information

Before https://github.com/OpenOrienteering/mapper/pull/1700/commits/918dd9b0df1cc752dd65e83c0c259585e946e557 (v0.9.4), all area symbols were created with rotatable patterns during OCD import. This worked for vineyards et al. That commit put the pattern rotation property into the object rotation flag. Maybe this is the wrong place.

Note that even OCD 12 format is affected, despite being the subject of investigation for GH-1677/GH-1700 which are the context of the commit.

**Comment by lpechacek** [comment]
Interesting. I'm seeing exactly the opposite. Rotation is enabled on objects where it's supposed to be disabled. There's something fishy and I'll take a closer look. Thanks for the report!

**Comment by dg0yt** [comment]
AFAIU there is no such bit for fill pattern rotation, and that's why the fill pattern rotation area symbol property was unconditionally set to true before 918dd9b. This is different from the point symbol rotation property.

**Comment by dg0yt** [comment]
To make an example: individual rotation can be set for marsh or undergrowth objects in OCAD?

**Comment by lpechacek** [comment]
> To make an example: individual rotation can be set for marsh or undergrowth objects in OCAD?

Yes. I tried to rotate the undergrowth area pattern first and then I had to take a short break to recover.
![ocad-demo](https://user-images.githubusercontent.com/5584406/113339720-33fc9480-932b-11eb-9fe4-21fe111f98c3.gif)

The north orientation comes into play during map rotation. There is a checkbox (default ON) that can be unchecked, and then patterns of area symbols with the north orientation set are not rotated, while all other area patterns change their angle. This is the essence of #1677 which I misunderstood on the first attempt.
![image](https://user-images.githubusercontent.com/5584406/113340107-c00ebc00-932b-11eb-8b2b-c82d35535648.png)

Replicated with OCAD 8.12 demo edition.

**Comment by dl3sdo** [comment]
Replicated with OCAD 12.

**Comment by dg0yt** [comment]
This one of few open issues currently assigned to the v0.9.6 milestone.
